src/riak_test_escript.erl

```diff
@@ -297,26 +297,45 @@ run_test(Test, Outdir, TestMetaData, Report, HarnessArgs, NumTests) ->
         1 -> keep_them_up;
         _ -> rt:teardown()
     end,
-    CoverageFile = rt_cover:maybe_export_coverage(Test, CoverDir, erlang:phash2(TestMetaData)),
+    CoverageFile = rt_cover:maybe_export_coverage(Test,
+                                                  CoverDir,
+                                                  erlang:phash2(TestMetaData)),
     case Report of
         undefined -> ok;
         _ ->
-            {value, {log, L}, TestResult} = lists:keytake(log, 1, SingleTestResult),
+            {value, {log, L}, TestResult} =
+                lists:keytake(log, 1, SingleTestResult),
             case giddyup:post_result(TestResult) of
                 error -> woops;
                 {ok, Base} ->
                     %% Now push up the artifacts, starting with the test log
                     giddyup:post_artifact(Base, {"riak_test.log", L}),
-                    [ giddyup:post_artifact(Base, File) || File <- rt:get_node_logs() ],
```

Comment:
Also, if you do find lines that are still too long, could you chop them up in a separate commit next time please? When it's all in one commit, it makes it tricky to identify what code has actually changed vs. what has just been reformatted.